### PR TITLE
Release 20210610

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] Correct Markdown and formatting
- [x] All content reviewed

-----
[View rendered _posts/2021-04-08-March-2021-newsletter.md](https://github.com/publiccodenet/blog/blob/release-20210610/_posts/2021-04-08-March-2021-newsletter.md)
[View rendered _posts/2021-06-08-newsletter-may-2021.md](https://github.com/publiccodenet/blog/blob/release-20210610/_posts/2021-06-08-newsletter-may-2021.md)
[View rendered _posts/2021-06-08-product-management-in-open-source.md](https://github.com/publiccodenet/blog/blob/release-20210610/_posts/2021-06-08-product-management-in-open-source.md)
[View rendered _posts/2021-06-10-episode-six-of-lets-talk-about-public-code.md](https://github.com/publiccodenet/blog/blob/release-20210610/_posts/2021-06-10-episode-six-of-lets-talk-about-public-code.md)

### CODE REVIEW

**Comment by ElenaFdR** [comment]
Broken links flagged by Travis load in the browser.

**Review by felixfaassen — APPROVED** [review]
Looks A ok. Ignoring Travis CI build errors as they are infra related there ;-)

File: _posts/2021-06-08-product-management-in-open-source.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Product management in open source: you are responsible!"
+date: 2021-06-08
+author: Felix Faassen
+type: blogpost
+excerpt: How is product management done in open source projects and how does it differ from product management in public code
```
Comment (Ainali) [inline _posts/2021-06-08-product-management-in-open-source.md:6]: No category?

**Review by Ainali — APPROVED** [review]

File: _posts/2021-06-08-product-management-in-open-source.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+categories:
+  - news
```
Comment (ElenaFdR) [inline _posts/2021-06-08-product-management-in-open-source.md:8]:
```suggestion
  - codebase stewardship
```

**Pull request merged**